CONTRIBUTING.md

```diff
@@ -62,8 +62,8 @@ Building and running the code
 Run the following steps, from the `src` directory created in the steps above:
 
  * `gclient sync` to update your dependencies.
- * `./sky/tools/gn --android` to prepare your build files.
- * `ninja -C out/android_Debug` to build an Android Debug binary.
+ * `./sky/tools/gn --android` to prepare your build files to build an Android Debug binary.
+ * `ninja -C out/android_Debug` to actually build an Android Debug binary.
```

Comment:
Use the GN --debug flag to build a debug engine

"gn --android --debug" outputs to out/android_develop_debug